packages/bpk-component-pagination/src/themeAttributes.js

```diff
@@ -0,0 +1,29 @@
+/*
+  * Backpack - Skyscanner's Design System
+  *
+  * Copyright 2017 Skyscanner Ltd
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *   http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+import {
+  secondaryThemeAttributes,
+} from 'bpk-component-button';
+
+export default [
+  ...secondaryThemeAttributes,
+  'paginationBackgroundColorSelected',
```

Comment:
These names do not follow our naming convention, see https://github.com/Skyscanner/backpack/blob/master/decisions/theme-props-naming.md

Reply:
I know, I'm a rebel! 😄 

I've used the name on the Jira ticket, I won't do that anymore

Reply:
just to confirm, is
```
  'paginationSelectedBackgroundColor',
  'paginationNudgerActiveColor',
  'paginationNudgerHoverColor',
  'paginationNudgerNormalColor',
```
correct? 
cc @shaundon

Reply:
Sounds good to me!

Reply:
Update: Sorry, what I *meant* to say was that the 'normal' bit is redundant and can be omitted. 😅